renderer/HardwareBaseGPU.cpp

```diff
@@ -528,6 +476,26 @@ void rend_DrawPolygon2D(int handle, g3Point **p, int nv) {
   gpu_RenderPolygon(&vArray[0], nv);
 }
 
+color_array DeterminePointColor(g3Point const* pnt, bool disableGouraud, bool checkTextureQuality, bool flatColorForNoLight) {
```

Comment:
the function is used in critical paths, consider inlining it with `inline`

Reply:
`inline` doesn't actually do what it's commonly thought to do: since c++11, it's a _linker_ flag indicating that multiple definitions are allowed, not a _compiler_ flag indicating that the function should be inlined into its callsites.

optimizers got smart enough (and LTO became enough of a thing) that the previous meaning of "plop this code directly in place instead of generating a callsite" became a bit useless

Reply:
I've been lied to! :smile: Thanks for the heads-up